spooky-oysters/React-FrontEnd
# km-error-fix-tp

```sh
git fetch --all
git checkout km-error-fix-tp
npm start
```
Browser URL: ```http://bangazon.com:3000/```
**Must have the API project running for the following to work:**

## Impacts
Customer.js - a new customer component
README.md
Update of Detail component to clear bug

## Testing ##
Should see customer component
1. List all current customers
1. Clicking on customer brings up detail
1. Add new brings up form
1. Fill in firstname and lastname then submit
1. Click the 'List Customers' button and you should see the new customer
